build.sbt

```diff
@@ -209,26 +209,15 @@ lazy val scalameta = crossProject
 lazy val scalametaJVM = scalameta.jvm
 lazy val scalametaJS = scalameta.js
 
-lazy val scalahost = project
-  .in(file("scalahost/core"))
-  .settings(
-    moduleName := "scalahost",
-    description := "Scala.meta semantic API integration for Scala 2.x (scalac).",
-    publishableSettings,
-    hasLargeIntegrationTests,
-    isFullCrossVersion,
-    libraryDependencies += "org.scala-lang" % "scala-compiler" % scalaVersion.value
-  )
-  .dependsOn(scalametaJVM)
-
 lazy val scalahostNsc = project
   .in(file("scalahost/nsc"))
   .settings(
-    moduleName := "scalahost-nsc",
```

Comment:
Also, even though both paradise and scalahost will bring their own copies of scalameta, that's not going to be a problem, because every compiler plugin has its own classloader.